static/templates/message_controls.hbs

```diff
@@ -9,12 +9,18 @@
     </div>
     {{/unless}}
 
+    {{#unless should_display_quote_and_reply}}
```

Comment:
`should_display_quote_and_reply` will be undefined here since it is not passed to the template. And thus this button will be shown always.